more/free-programming-cheatsheets.md

```diff
@@ -259,6 +260,14 @@
 * [Scapy Cheat Sheet](https://www.templateroller.com/template/160817/scapy-cheat-sheet-jeremy-stretch.html) - Jeremy Stretch (PDF)
 
 
+#### Django
+
+* [All in one Cheatsheet of Django](https://cheatography.com/ogr/cheat-sheets/django/pdf/) - Cheatography (PDF)
+* [Beginner's Python Cheat Sheet – Django](https://edu.anarcho-copy.org/Programming%20Languages/Python/Python%20CheatSheet/beginners_python_cheat_sheet_pcc_django.pdf) - Eric Matthes (PDF)
```

Comment:
This comes from https://ehmatthes.github.io/pcc_2e/cheat_sheets/cheat_sheets/ and that's what we should link. Please change.

Reply:
Done at 9b78d4564852b8ccf25bc3d1a16aaca2fc3e5910